webapp/components/Editor/ContentViewer.vue

```diff
@@ -39,3 +39,10 @@ export default {
   },
 }
 </script>
+<style lang="scss">
+.ProseMirror h3,
+.ProseMirror h4,
+.ProseMirror hr {
+  margin: 24px 0 8px;
+}
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-04T14:08:14Z" title="Tuesday, August 4th 2020, 4:08:14 pm +02:00">Aug 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -39,3 +39,10 @@ export default {
   },
 }
 </script>
+<style lang="scss">
+h3,
+h4,
+hr {
+  margin: 8px 0;
+}
```


Would be cool if you could apply these as a class to the HTML tag, see:
https://github.com/Human-Connection/Human-Connection/blob/master/webapp/scss.md

Reply:
> [<img alt="ritesh-pandey" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ritesh-pandey) **Authored by [ritesh-pandey](https://github.com/ritesh-pandey)**
_<time datetime="2020-08-04T16:01:08Z" title="Tuesday, August 4th 2020, 6:01:08 pm +02:00">Aug 4, 2020</time>_

---


Aah! I missed that.

Reply:
> [<img alt="ritesh-pandey" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ritesh-pandey) **Authored by [ritesh-pandey](https://github.com/ritesh-pandey)**
_<time datetime="2020-08-05T08:13:21Z" title="Wednesday, August 5th 2020, 10:13:21 am +02:00">Aug 5, 2020</time>_

---


This markup is generated by `tiptap`. To add custom class to tag, there should be some mechanism to pass the custom class name via `props`. We will have to dig into TipTap documentation for that. I haven't explored that.

Otherwise, we can increase specificity by using expression 
```
.ProseMirror h3,
.ProseMirror h4,
.ProseMirror hr {
  margin: 24px 0 8px;
}
```
Similar style is followed by `ProseMirror`